### ISSUE TITLE
[Northrend] [Raids] Obsidian Sanctum : Sartharion

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3920]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 After me and a couple members of the raid group entered the portal in p3 and came out of it after killing the add, we could not see the boss or attack him. /Target displayed his icon tho.

**Expected behaviour**: Tell us what should happen instead
 After the players enter the portal in p3, and then leave it, they should see the raid boss Sartharion and be able to attack him.

**Steps to reproduce the problem**:

1.  Enter a raid group.
2.  Reach p3.
3.  Enter the portal/leave the portal.
4.  Check if you can see the boss/attack him.

**Include proofs for this behaviour**

https://postimg.org/image/xpkesxf5d/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Works fine for me on ptr, maybe some internal dalaran-wow updates break his mechanic

https://github.com/TrinityCore/TrinityCore/blob/a249d86b00eb0854cb925f8bea76ddbb364208df/src/server/scripts/Northrend/ChamberOfAspects/ObsidianSanctum/boss_sartharion.cpp

**Comment by TeridanDW** [comment]
Can't reproduce the issue on ptr also, if this is still happening please open a new report.
